### PR TITLE
gui from jyothiraditya

### PR DESCRIPTION
**Pull request by jyoth777**
basic gui

password generator

### CODE REVIEW

**Comment by jyoth777** [comment]
i am sorry i didn't see what you had sent on whatsapp

File: gui.py [hunk 1]
```diff
@@ -1,15 +1,67 @@
...
+#root window 
+root=Tk()
+root.title("Password Generator")
+root.geometry("900x600")
 
-# Add a label to display the generated password
 
-# Add checkboxes to select the characters to be used
+bg=ImageTk.PhotoImage(Image.open("white.jpg"))
+l1=Label(root,image=bg).place(x=0,y=0,relwidth=1,relheight=1)
```
Comment (Shifaan0010) [inline gui.py:14]: Not needed since the background is already white

File: gui.py [hunk 2]
```diff
@@ -1,15 +1,67 @@
...
+
+def clearFrame():
+    # destroy all widgets from frame
+    for widget in frame.winfo_children():
+       widget.destroy()   
 
 def button_pressed():
-    '''Function that is called when the button "Generate" is clicked'''
+ clearFrame()
+ random_pass(length=int(slider.get()),uppercase=int(a.get()),lowercase=int(b.get()),punctuations=int(c.get()),numbers=int(d.get()))
```
Comment (Shifaan0010) [inline gui.py:55]: Not sure why you're deleting the label here and then creating a new one in the random_pass() function

You could just modify the text in the label

File: gui.py [hunk 3]
```diff
@@ -1,15 +1,67 @@
...
+
+#generate button which generates random password when clicked 
+button=Button(root,text="generate",image=button_image,command=button_pressed,borderwidth=0,bg='#ffffff').pack()
+
+
+frame=LabelFrame(root,bg='#ffffff',borderwidth=0,highlightthickness=0)
+frame.pack(pady=10)
```
Comment (Shifaan0010) [inline gui.py:62]: You could do this without a frame

**Review by Shifaan0010 — COMMENTED** [review]
Looks good.
You haven't uploaded the images used

File: gui.py (hunk 2 above)
Comment (jyoth777) [reply gui.py]: I'm new programming so bear with  stupid mistakes

**Comment by jyoth777** [comment]
generate3:
generate3

exit1:
exit1

File: gui.py (hunk 2 above)
Comment (Shifaan0010) [reply gui.py]: I've fixed it now

Comment (Shifaan0010) [reply gui.py]: That's fine. Don't worry about it

**Comment by Shifaan0010** [comment]
Wrong branch. Merge it with the branch gui

**Pull request closed without merging**